constants/appLinks.ts

```diff
@@ -57,6 +57,79 @@ const IBM_Q_EXPERIENCE: GeneralLink = {
   label: 'IBM Quantum Lab'
 }
 
+interface Prerequisite {
+  title: string;
+  description: string;
+  segment: CtaClickedEventProp;
+  url: string;
+}
+
+const introductionCoursePrerequisites: Prerequisite[] = [
+  {
+    title: 'Setting Up Your Environment',
+    description:
+      'This is a comprehensive guide for setting up your environment on your personal computer for working with Qiskit Textbook.',
+    segment: {
+      cta: 'setting-up-your-environment', location: 'prerequisite-material'
+    },
+    url:
+      'https://learn.qiskit.org/course/ch-prerequisites/environment-setup-guide-to-work-with-qiskit-textbook'
+  },
+  {
+    title: 'Python and Jupyter Notebooks',
+    description:
+      "Python is a programming language where you don't need to compile. You can just run it line by line...",
+    segment: {
+      cta: 'python-and-jupyter-notebooks', location: 'prerequisite-material'
+    },
+    url:
+      'https://learn.qiskit.org/course/ch-prerequisites/introduction-to-python-and-jupyter-notebooks'
+  }
+]
+
+const summerSchool2021Prerequisites: Prerequisite[] = [
```

Comment:
I think that it's better to have this `summerSchool2021Prerequisites` on the `quantum-computing-and-quantum-learning-2021` page.

Also, if in the future we have repeated prerequisites, we could think in a file for having them and not repeat code